tools/afl/src/scell_seal_decrypt.c

```diff
@@ -23,38 +23,51 @@
 
 #include "readline.h"
 
-int main(void)
+int main(int argc, char **argv)
 {
 	themis_status_t err = THEMIS_SUCCESS;
 
 	/*
 	 * Read test data.
 	 */
 
+	if (argc != 2)
+	{
+		return 1;
+	}
+
+	FILE* input = fopen(argv[1], "rb");
+	if (!input)
+	{
+		return 1;
```

Comment:
what about to print some message that we expect filepath in one required argument to avoid the need to dive into sources to understand why exit status 1?

Reply:
TBH, the exit statuses are meaningless now. AFL ignores them completely, it's in only for the crashes. I hoped that it might discriminate between different exit codes, but nope.

You're right. I guess I'll add some messages to stderr when we exit with non-zero status. This should make it easier for humans with the tools.

Reply:
@Lagovas here you go. Now the tools print a "usage" line if the command-line is incorrect, as well as some comments on why exactly we're exiting with non-zero status or aborting.